src/platform_impl/linux/x11/mod.rs

```diff
@@ -309,9 +309,11 @@ impl<T: 'static> EventLoop<T> {
             }
             // Empty the redraw requests
             {
-                while let Ok(wid) = self.redraw_channel.try_recv() {
+                while let Ok(window_id) = self.redraw_channel.try_recv() {
                     sticky_exit_callback(
-                        Event::RedrawRequested(crate::window::WindowId(super::WindowId::X(wid))),
+                        Event::RedrawRequested(crate::window::WindowId(super::WindowId::X(
+                            window_id,
+                        ))),
```

Comment:
Oh now I know why you used `wid`. :smiley: 

Generally I'd recommend to extract things into variables when ugly wrapping like this is introduced, using full paths from the crate root also certainly doesn't help with making things shorter.

That's up to you as the maintainer of the X11 backend though.